project.janet

```diff
@@ -7,5 +7,16 @@
   :repo "git+https://github.com/AlecTroemel/junk-drawer"
   :dependencies ["spork"])
 
+
+(add-loader)
+(import /src/cache)
+
+(task "cache.c" []
+      (cache/render "cache.c"))
```

Comment:
could we put the rendered C file in `src/`, instead of the root of the project?

Reply:
Indeed. It should be ignored anyway.